doc/whats_new/v1.5.rst

```diff
@@ -111,6 +111,8 @@ Changelog
   :class:`~metrics.PrecisionRecallDisplay`, :class:`~metrics.DetCurveDisplay`,
   :class:`~calibration.CalibrationDisplay`.
   :pr:`28051` by :user:`Pierre de Fréminville <pidefrem>`.
+- |Fix|:class:`metrics.classification_report` now shows only accuracy and not micro-average when input is a subset of labels.
```

Comment:
This need to be moved in 1.4 under the 1.4.2 changelog. We will include it in the next bug fix release.

Reply:
You need to make sure that you entry is less than 88 characters also in the changelog.